### PR TITLE
Updates license, notice, readme and contributing

### PR DESCRIPTION
**Pull request by nvollmar**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
                                  Apache License
                            Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+                        https://www.apache.org/licenses/
```
Comment (pjfanning) [inline LICENSE:3]: due to https://issues.apache.org/jira/browse/LEGAL-630, can you keep the http:// links?

File: NOTICE [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Apache Pekko Persistence Cassandra
```
Comment (pjfanning) [inline NOTICE:1]: Should say Pekko Management

**Comment by nvollmar** [comment]
Fixed

File: LICENSE (hunk 1 above)
Comment (nvollmar) [reply LICENSE]: Created a PR in pekko-persistence-cassandra to fix that as well

**Review by pjfanning — APPROVED** [review]
lgtm

**Pull request merged**